setup.cfg

```diff
@@ -0,0 +1,4 @@
+[nosetests]
+tests = test/unit,
+        utils,
+        roles/openshift_master_facts/test,
```

Comment:
@rhcarvalho I'm not sure how well that would work, since then the test would be in the Ansible module lookup path.